pkg/util/names.go

```diff
@@ -1,4 +1,4 @@
-package kube
+package util
```

Comment:
OOI, what is the motivation to move package? 

On that note, there is a k8s.go in the util package. It contains an `EncodeKubernetesName` which does not even seem to be used. What's about moving all the functions in this file into there (potentially removing `EncodeKubernetesName`) on the go?

names.go is very generic and when just looking at file names in a package. I somehow find `k8s.go` gives me a better idea on what to expect, leaving potentially the door open for none name related utility functions.

Reply:
Circular dependency hell, basically. That said, I'll move it into `k8s.go`.

Reply:
I see.